x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/api.ts

```diff
@@ -75,22 +73,25 @@ export const fetchRules = async ({
 
   const endpoint =
     id != null
-      ? `${chrome.getBasePath()}${DETECTION_ENGINE_RULES_URL}?id="${id}"`
-      : `${chrome.getBasePath()}${DETECTION_ENGINE_RULES_URL}/_find?${queryParams.join('&')}`;
+      ? `${DETECTION_ENGINE_RULES_URL}?id="${id}"`
+      : `${DETECTION_ENGINE_RULES_URL}/_find?${queryParams.join('&')}`;
```

Comment:
Seeing a `404` for this request -- looks like the `?` is being encoded now.

Request URL from dev tools network tab:

`http://localhost:5601/api/detection_engine/rules/_find%3Fpage=1&per_page=20&sort_field=enabled&sort_order=desc`

Reply:
@spong good catch, I'll fix that. It also looks like we're no longer using the `id` form of this function (and instead using `fetchRuleById` ?). I'm going to delete that logic branch here also unless you say otherwise.

Reply:
Yeah, that sounds good. I added this functionality in one branch while @XavierM had `fetchRuleById` in an other, so that's how this happened. Makes sense to remove this logic for a better separation of concerns  -- thanks! 🙂